lib/chef/knife/vault_list.rb

```diff
@@ -35,7 +35,7 @@ def run
         bags.each_key do |bagname|
           vaultbags.push(bagname) if bag_is_vault?(bagname)
         end
-        output vaultbags.join("\n")
```

Comment:
Isn't this going to result in hard to read output on the CLI?

Reply:
Hi @tas50, nothing impacted on the CLI's output, but if we use join it is causing issues for JSON format, as posted in the issue.